File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entity/PersianCalendarEvent.java
package com.byagowi.persiancalendar.entity;

import calendar.PersianDate;

public class PersianCalendarEvent extends AbstractEvent<PersianDate> {
    public PersianCalendarEvent(PersianDate date, String title, boolean holiday) {
        this.date = date;
        this.title = title;
        this.holiday = holiday;
    }
}

File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entity/PersianCalendarEvent.kt
package com.byagowi.persiancalendar.entity

import calendar.PersianDate

/**
 * PersianCalendarEvent
 *
 * @author ebraminio
 */
class PersianCalendarEvent(val date: PersianDate, title: String,
                           holiday: Boolean) : AbstractEvent() {

    init {
        this.title = title
        this.holiday = holiday
    }
}